<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Dell123" ignored="0" name="Test" tests="2" failures="2" timestamp="2023-04-26T00:15:28 IST" time="11.418" errors="0">
  <testcase name="identifyLogoField" time="11.332" classname="Listeners.Login">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Listeners.Login.identifyLogoField(Login.java:29)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- identifyLogoField -->
  <testcase name="identifyLogoField" time="11.355" classname="Listeners.Login">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at PracticeScenario.BaseTestutil.screenShot(BaseTestutil.java:107)
at Listeners.Listeners.onTestFailure(Listeners.java:28)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- identifyLogoField -->
</testsuite> <!-- Test -->
